### PR TITLE
here's some stuff and things

### PR DESCRIPTION
**Pull request by extratoristrial**

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 export default function Footer(){
     return `
     <div id="footer">
+
```
Comment (NAlexPear) [inline components/Footer.js]: Make sure you aren't unnecessarily padding your components with extra newlines

File: components/Navigation.js [hunk 2]
```diff
@@ -1,6 +1,16 @@
 export default function Navigation(){
     return `
     <div id="navigation">
+
+    <ul>
```
Comment (NAlexPear) [inline components/Navigation.js]: Make sure that your children are indented by one level from their parents!

File: index.html [hunk 3]
```diff
@@ -6,9 +6,12 @@
         <!-- CDN reset -->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
 
-        <!-- linked stylesheet below -->
-        <link rel="stylesheet" href="css/style.css">
-    </head>
+    <!-- linked stylesheet below -->
+    <link rel="stylesheet" href="css/style.css">
+
+    <title>SavvyCoders Books, LLC.</title>
```
Comment (NAlexPear) [inline index.html]: There's already a `<title>` tag on line 5. Please modify that one instead of adding these!

**Comment by extratoristrial** [comment]
changes have been made dear leader

File: components/Header.js [hunk 4]
```diff
@@ -1,6 +1,10 @@
 export default function Header(){
     return `
     <div id="header">
+
+        <h1>SavvyCoders Books</h1>
```
Comment (NAlexPear) [inline components/Header.js]: There's still some padding here, too! Plz remove

File: components/Navigation.js [hunk 5]
```diff
@@ -1,6 +1,14 @@
 export default function Navigation(){
     return `
     <div id="navigation">
+    <ul>
```
Comment (NAlexPear) [inline components/Navigation.js]: This `ul` should be indented relative to the `div`

**Comment by extratoristrial** [comment]
YOUR REQUEST IS GRANTED DEAR LEADER

File: index.js [hunk 6]
```diff
@@ -41,4 +41,8 @@ var book2 = {
         'High Times Best Seller',
         'Each Chapter Contains a Different Strain Thoughts - Seth Rogen'
     ]
+<<<<<<< HEAD
```
Comment (NAlexPear) [inline index.js]: MERGE CONFLICT OH NOES

**Pull request merged**